.openpublishing.publish.config.json

```
@@ -36,6 +36,12 @@
       "url": "https://github.com/Microsoft/templates.docs.msft.pdf",
       "branch": "main",
       "branch_mapping": {}
+    },
+    {
+      "path_to_root": "microsoft-edge/webview2/code/",
```

Can we include the word sample in here somewhere in case we want to add other repositories in the future?
```suggestion
      "path_to_root": "microsoft-edge/webview2/code/sample/",
```